src/otx/core/model/module/base.py

```diff
@@ -133,12 +134,32 @@ def state_dict(self) -> dict[str, Any]:
         state_dict["meta_info"] = self.meta_info
         return state_dict
 
-    def load_state_dict(self, state_dict: dict[str, Any], *args, **kwargs) -> None:
+    def _load_from_prev_otx_ckpt(self, state_dict: dict[str, Any]) -> dict[str, Any]:
+        """Attach the model.model prefix to load without problem."""
+        msg = "Trying to load the model checkpoint created by OTX 1.X. it will be converted to OTX2.0 format."
```

Comment:
Any idea, for how long are we going to support this backward compatibility? If it is temporary, it might also be an idea to add a deprecation warning for future releases?

Reply:
For the backward compatibility, as far as I know, we need to wait for the decision from Geti staked holders. Maybe it isn't just temporary.